src/components/dell/DellLoginPage/DellLoginPage.tsx

```diff
@@ -168,6 +167,17 @@ export function DellLoginPage({
             </StyledWrapper>
             <StyledWrapper>
                 <form className={isLoginFormSubmitted ? componentClasses.form.validated : ''}>
+                    {error && (
+                        <MessageBar
+                            variant="error"
+                            className={getMarginClassName('bottom', 3)}
+                            dismissible={false}
+                            open={error !== null}
```

Comment:
`error` can't be `null` in this line if I'm not mistaken, so maybe: 
```suggestion
                            open
```

Reply:
changed it: https://github.com/cloudify-cosmo/cloudify-ui-components/pull/244/commits/ad1a9b91ab8b1ef1b082745cee43c2c3f02c8dd5